packages/video_player/video_player_web/example/integration_test/video_player_web_test.dart

```diff
@@ -125,6 +125,16 @@ void main() {
       expect(VideoPlayerPlatform.instance.pause(await textureId), completes);
     });
 
+    testWidgets('can enter fullscreen', (WidgetTester tester) async {
+      expect(VideoPlayerPlatform.instance.enterFullScreen(await textureId),
+          completes);
```

Comment:
This is only testing that the future completes, not that it does anything. Wouldn't at empty implementation of `enterFullScreen` pass this test?